src/test/java/org/fundacionjala/sfdc/stepdefinition/ContactSteps.java

```diff
@@ -0,0 +1,164 @@
+package org.fundacionjala.sfdc.stepdefinition;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.commons.PropertiesManager;
+import org.fundacionjala.sfdc.pageobjects.Contacts.SFCDetailsPage;
+import org.fundacionjala.sfdc.pageobjects.Contacts.SFCMainPage;
+import org.fundacionjala.sfdc.pageobjects.Contacts.SFCNewModifyPage;
+import org.fundacionjala.sfdc.pageobjects.SalesForceContacts;
+import org.fundacionjala.sfdc.pageobjects.SalesForceMainTabClassic;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+
+import java.util.Properties;
+import java.util.StringJoiner;
+
+/**
+ * ContactSteps.java
+ * Class for steps of account.feature.
+ */
+public class ContactSteps {
+    private WebDriver webDriver;
+    private SalesForceContacts salesForceContacts;
+    private Properties propertiesManager;
+    private SFCDetailsPage contactsDetailPage;
+    private SFCMainPage contactsMainPage;
+    private SFCNewModifyPage contactsNewModifyPage;
+
+
+    /**
+     * Given step.
+     */
+    @Given("^Go to contacts section$")
+    public void goToContactsSection() {
+        System.out.println("This Step goes to contacts section");
+        SalesForceMainTabClassic salesForceMainTab;
+        salesForceMainTab = new SalesForceMainTabClassic("a.contactBlock", "li#AllTab_Tab");
+        salesForceMainTab.displayOptions();
+        salesForceMainTab.goToAccounts();
+
+    }
+
+    /**
+     * When step.
+     */
+    @When("^I press new contacts button a new form is displayed$")
+    public void IPressNewContactsButtonANewFormIsDisplayed() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)